platform-input-support.py

```diff
@@ -25,10 +30,16 @@ def main():
     logger.debug("Get command line arguments")
     args = cfg.get_args()
     logger.debug("Prepare YAML reader for configuration file")
-    yaml = YAMLReader(args.config)
-    logger.debug(f"Read configuration file")
-    yaml_dict = yaml.read_yaml()
-    print_list_steps(yaml.get_list_keys())
+    try:
+        yaml = YAMLReader(args.config)
+        logger.debug("Read configuration file")
+        yaml_dict = yaml.read_yaml()
+        print_list_steps(yaml.get_list_keys())
+    except YAMLReaderException as err:
+        message = f"When trying to read the YAML config, \
```

Comment:
I would include the path details of the configuration file that the pipeline tried to read